core/options.c

```diff
@@ -2443,6 +2442,8 @@ options_init()
     /* .lspdata pages start out writable so no unprotect needed here */
     write_lock(&options_lock);
     ASSERT(sizeof(dynamo_options) == sizeof(options_t));
+    /* Set to default options. */
```

Comment:
This changes the behavior of `options_init`. Maybe add a comment to the documentation of the function to make it clear that all previously set options will be overwritten by the default values?